demisto_sdk/dev_tools/lint_manager.py

```diff
@@ -0,0 +1,139 @@
+            fail_pkgs = []
+            good_pkgs = []
+
+            # run CommonServer non parallel to avoid conflicts
+            # when we modify the file for mypy includes
+            if 'Scripts/CommonServerPython' in pkgs_to_run:
+                pkgs_to_run.remove('Scripts/CommonServerPython')
+                res, _ = self._run_single_package_thread(package_dir='Scripts/CommonServerPython')
+                if res == 0:
+                    good_pkgs.append('Scripts/CommonServerPython')
+
+                else:
+                    fail_pkgs.append('Scripts/CommonServerPython')
+
+            with concurrent.futures.ThreadPoolExecutor(max_workers=max_workers) as executor:
+                futures_submit = [executor.submit(self._run_single_package_thread, dir) for dir in pkgs_to_run]
+                for future in list(concurrent.futures.as_completed(futures_submit)):
+                    result = future.result()
+                    status_code = result[0]
+                    package_ran = result[1]
+                    if status_code == 0:
+                        good_pkgs.append(package_ran)
+
+                    else:
+                        fail_pkgs.append(package_ran)
+
+                if fail_pkgs:
+                    print_color("\n******* FAIL PKGS: *******", LOG_COLORS.RED)
+                    print_color("\n\t{}\n".format("\n\t".join(fail_pkgs)), LOG_COLORS.RED)
+
+                if good_pkgs:
+                    print_color("\n******* SUCCESS PKGS: *******", LOG_COLORS.GREEN)
+                    print_color("\n\t{}\n".format("\n\t".join(good_pkgs)), LOG_COLORS.GREEN)
+
+                if not good_pkgs and not fail_pkgs:
+                    print_color("\n******* No changed packages found *******\n", LOG_COLORS.YELLOW)
+
+                if fail_pkgs:
+                    sys.exit(1)
+
+    def _should_run_pkg(self, pkg_dir: str) -> bool:
+        diff_compare = os.getenv("DIFF_COMPARE")
+        if not diff_compare:
+            return True
+        if os.getenv('CONTENT_PRECOMMIT_RUN_DEV_TASKS'):
+            # if running in precommit we check against staged
+            diff_compare = '--staged'
+        res = subprocess.run(["git", "diff", "--name-only", diff_compare, "--", pkg_dir], text=True,
+                             capture_output=True)
+        if res.stdout:
+            return True
+        return False
+
+    def _run_single_package_thread(self, package_dir) -> Tuple[int, str]:
+        linter = Linter(package_dir, no_test=not self.run_args['tests'],
+                        no_pylint=not self.run_args['pylint'], no_flake8=not self.run_args['flake8'],
+                        no_mypy=not self.run_args['mypy'], verbose=self.log_verbose, root=self.root,
+                        keep_container=self.keep_container, cpu_num=self.cpu_num, configuration=self.configuration)
+
+        return linter.run_dev_packages(), package_dir
+
+    @staticmethod
+    def add_sub_parser(subparsers):
```

Comment:
where is this being used?

Reply:
in the core.py part - this defines all flags that are used with Lint.